saleor/tests/e2e/checkout/utils/checkout_create.py

```diff
@@ -45,6 +45,31 @@
         isPrivate
         clickAndCollectOption
       }
+      lines {
+        id
+        totalPrice {
+          gross {
+            amount
+          }
+          net {
+            amount
+          }
+          tax {
+            amount
+          }
+        }
+        undiscountedTotalPrice {
+          amount
+        }
+        unitPrice {
+          net {
```

Comment:
@szczecha Shouldn't we return gross value here as well?

Reply:
thanks, fixed: https://github.com/saleor/saleor/pull/13649/commits/57c854d9aac92c419c262f145f67c0d3042b16d7